universal-application-tool-0.0.1/app/assets/javascripts/azure_delete.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * This is responsible for deleting a file that was uploaded to Azure blob storage.
+ */
+
+class AzureDeleteController {
+  constructor() {
+    const deleteContainer = document.getElementById('fileupload-delete-button')
+    deleteContainer.addEventListener('click', (event) => this.attemptDelete())
+  }
+
+  attemptDelete() {
+    const azblob = window['azblob']
```

Comment:
suggestion (non-blocking): can this be loaded once in the constructor or globally outside the class instead of in the functions?

Reply:
I loaded it in the constructor and passed it as a function param, let me know what you think. I have it as "any" because I could not for the life of me figure out the type, if you happen to know I will update it!